database/database.go

```diff
@@ -301,6 +360,87 @@ func (db *Database) Close() {
  * Private
  **********************************************************************************************************************/
 
+func (db *Database) getDataFromQuery(query string, queryParams []any, result ...interface{}) error {
```

Comment:
1. Use either any or interface{}
2. If you introduce generic function, existing API should be changed to use it as well ;)

Reply:
2. Done